### PR TITLE
94 task list page

### PR DESCRIPTION
**Pull request by ricardovdheijden**
#94 

Adding task list, which is a different view on of the processesList. This PR does a small refactor on the processesList component to make it reusable in the tasks page.

### CODE REVIEW

File: apps/wfo-ui/pages/processes/index.tsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import React from 'react';
-import { WFOProcessListPage } from '@orchestrator-ui/orchestrator-ui-components';
+import { WFOProcessesListPage } from '@orchestrator-ui/orchestrator-ui-components';
 
 export default function ProcessesPage() {
-    return <WFOProcessListPage />;
+    return <WFOProcessesListPage />;
```
Comment (tjeerddie) [inline apps/wfo-ui/pages/processes/index.tsx:5]: what was wrong with `WFOProcessListPage`?

I believe it was correctly named

File: packages/orchestrator-ui-components/src/pages/tasks/WFOTaskListPage.tsx [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+            value: 'true',
+        },
+        {
+            field: 'status',
+            value: 'running-failed-api_unavailable-inconsistent_data',
+        },
+    ];
+
+    // Changing the order of the keys, resulting in an updated column order in the table
+    const handleOverrideTableColumns: (
```
[the remark this answers is not in the document]
Comment (ricardovdheijden) [reply packages/orchestrator-ui-components/src/pages/tasks/WFOTaskListPage.tsx:52]: The issue here is that the processesList have a defined order which is correct by itself. In the tasklist we want a different column order.

[the remark this answers is not in the document]
Comment (tjeerddie) [reply packages/orchestrator-ui-components/src/pages/tasks/WFOTaskListPage.tsx:52]: in `defaultTableColumns` should `process_id` be at the bottom of the order?

File: apps/wfo-ui/pages/processes/index.tsx (hunk 1 above)
Comment (ricardovdheijden) [reply apps/wfo-ui/pages/processes/index.tsx]: Yes, you're right. Done

File: packages/orchestrator-ui-components/src/pages/tasks/WFOTaskListPage.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (ricardovdheijden) [reply packages/orchestrator-ui-components/src/pages/tasks/WFOTaskListPage.tsx:52]: ProcessId is not the last one, 2 more props are below. But indeed, the defaultTableColumns in the ProcessList component contains the proper order for the process-list view

**Review by tjeerddie — APPROVED** [review]

**Pull request merged**